infra/main.tf

```diff
@@ -45,7 +45,7 @@ resource "google_storage_bucket_iam_member" "default" {
 }
 
 resource "google_storage_bucket_object" "icons" {
-  for_each     = fileset("${path.module}/google-cloud-icons", "*.svg")
+  for_each     = fileset("${path.root}/google-cloud-icons", "*.svg")
```

Comment:
question: Could you explain the use of path.root a bit more? I'm not sure how these pieces work.

Based on this change, I assume that path.module was updating to examples/simple_example when run via CI, but path.root is more consistent about references this `infra/` directory.

Reply:
[Hashicorp docs](https://developer.hashicorp.com/terraform/language/expressions/references#filesystem-and-workspace-info)

`path.root` uses the path of the root module configuration (in this case `infra/`).  I changed it from `path.module` which refers to the path of where the module is being referenced (in the case of int tests, `infra/examples/simple_example` which does not have the svg files referenced here.)